openforcefields/offxml/openff-2.0.0-rc.1.offxml

```diff
@@ -0,0 +1,376 @@
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+	<Author>The Open Force Field Initiative</Author>
+	<Date>2021-06-07</Date>
+	<Constraints version="0.3">
+		<Constraint smirks="[#1:1]-[*:2]" id="c1"/>
+		<!-- TIP3P Water -->
+		<Constraint smirks="[#1:1]-[#8X2H2+0:2]-[#1]" id="c1" distance="0.9572 * angstrom"/>
+		<Constraint smirks="[#1:1]-[#8X2H2+0]-[#1:2]" id="c2" distance="1.5139006545247014 * angstrom"/>
```

Comment:
@SimonBoothroyd For the "real" release, I wonder if we want to give the TIP3P constraints distinct IDs (maybe `HB-c1` for the general hbonds and `TIP3P-c1` and `2` for the TIP3P ones)